js/output/sql/sql-output.js

```diff
@@ -102,6 +102,14 @@ window.SQLOutput = Backbone.View.extend({
             const colName = errorMessage.split(ambColStr)[1].trim();
             errorMessage = i18n._("Ambiguous column name \"%(colName)s\".",
                 {colName});
+            // Note(danielhollas): Added a more helpful text as a separate string
+            // to preserve existing translations.
+            // I18N: This Oh Noes message follows "Ambiguous column name" SQL error
+            // I18N: Keep the space at the beginning of string.
+            errorMessage += " " + i18n._("Multiple tables that you're joining " +
+               "contain a column with that name. To use that column in your query, " +
+               "specify the table of the column. Example: firstTable.\"%(colName)s\"",
```

Comment:
Hm, seeing it in the screenshot, the example is throwing me off. It's weird for me to see a generic table name (firstTable) and then a specific column name (id). It's halfway between being an example and being the real thing.
 
Another idea is this sentence instead of "To use..."
"Specify the name of the table that contains the column, using the format "tableName.columnName".
(I'm not sure we need "to use that column in your query", since the error implies they are trying to use it, right?")
What do you think? What's most helpful?

Reply:
Agreed, I was not very satisfied either. I'll change it tomorrow. We do have access to the SQL statement that throws the error so we could try to extract at least one table name with some clever regexing. LMK if you think I should pursue that.

Reply:
I don't think it's worth it, since it's possible they mean the other table, right?

Reply:
@pamelafox right, good point, that would actually perhaps make it more confusing. I updated the wording and rebased on master, see comment below. Thanks!